app/components/login_button_component.html.erb

```diff
@@ -0,0 +1,7 @@
+<%= content_tag(
+      :button,
+      **tag_options,
+      class: css_class,
+    ) do %> 
+  Sign in with <%= content_tag(:span, APP_NAME, class: logo_class) %>
```

Comment:
Following previous comment, we can hard code the logo class since it's static:

```suggestion
  Sign in with <%= content_tag(:span, APP_NAME, class: 'login-button__logo') %>
```